applications/mne_analyze/plugins/sampleplugin/sampleplugin.cpp

```diff
@@ -0,0 +1,165 @@
+//=============================================================================================================
+/**
+ * @file     sampleplugin.cpp
+ * @author   Lorenz Esch <lesch@mgh.harvard.edu>;
```

Comment:
fix authors, version, year and copyright. check other files (cpp, h, pro) as well.